### ISSUE TITLE
A readonly-select can be changed

### ISSUE THREAD

**Issue by Reggino** [mozilla-services/react-jsonschema-form#524]
### Prerequisites

- [x] I have read the [documentation](https://github.com/mozilla-services/react-jsonschema-form/blob/master/README.md#readme);
- [x] In the case of a bug report, I understand that providing a [SSCCE](http://sscce.org/) example is tremendously useful to the maintainers.
### Description

A select-box with uiSchema ```{ "'ui:readonly": true  }``` can still be changed.

### Steps to Reproduce

Use schema:
```js
{
  type: "object",
  properties: {
    gender: {
      type: "string",
      enum: ["M", "F"],
      enumNames: ["Male", "Female"],
    },
  },
}
```
use UISchema:

```js
{
  gender: {
    "ui:readonly": true,
  },
}
```

And try changing the select-value.

#### Expected behavior

Not changable

#### Actual behavior

changable

### Version

0.44.0

**Comment by n1k0** [comment]
It seems the `readonly` attribute isn't supported for `<select/>` elements, where `disabled` should be used.

**Comment by n1k0** [comment]
Note that this would be fixed by #470 which has been neglected for too long.

**Comment by n1k0** [comment]
Fixed by #525.
